README.md

```diff
@@ -91,31 +106,44 @@ The final application structure (inside the Sources folder) looks like this:
 ├── Setup
 │   ├── app.swift
 │   ├── boot.swift
+│   ├── commands.swift
 │   ├── configure.swift
+│   ├── content.swift
+│   ├── databases.swift
+│   ├── middlewares.swift
 │   ├── migrate.swift
+│   ├── repositories.swift
 │   └── routes.swift
 ├── Utils
 ```
 
-# Configuration 
-Configuring your application correctly is one of the most important parts of a successful Vapor application. The main function of the configuring a Vapor application is correctly registering all of your services and 3rd party providers. 
+# Configuration
+
+Configuring your application correctly is one of the most important parts of a successful Vapor application. The main function of the configuring a Vapor application is correctly registering all of your services and 3rd party providers.
 
 **Note**: For more information on registering credentials and secrets, see the “Credentials” section.
 
 ### Files
+
 There are 6 files you should have:
 
 - app.swift (use the default template version)
 - boot.swift (use the default template version)
+- commands.swift // Optional
```

Comment:
Tiny detail - can we use parenthesis here instead of a comment-like syntax?

Reply:
of course... my bad